### PR TITLE
Ignore missing or unloadable PDBs

### PR DESCRIPTION
**Pull request by rzhao271**
It seems that the internal Electron builds with an 8kB page size, which isn't supported yet by BinSkim. In turn, BinSkim emits PDB load errors.

I also added a step that runs ls just to verify the PDBs are actually there.

This PR ignores errors related to missing PDBs or PDBs that fail to load.

### CODE REVIEW

File: build/gulpfile.scan.js [hunk 1]
```diff
@@ -46,8 +46,7 @@ BUILD_TARGETS.forEach(buildTarget => {
 	if (platform === 'win32') {
 		tasks.push(
 			() => electron.dest(destinationPdb, { ...config, platform, arch: arch === 'armhf' ? 'arm' : arch, pdbs: true }),
-			util.rimraf(path.join(destinationExe, 'swiftshader')),
```
Comment (rzhao271) [inline build/gulpfile.scan.js:49]: These symbols don't seem to show up anymore in the first place, ref https://dev.azure.com/monacotools/a6d41577-0fa3-498e-af22-257312ff0545/_apis/build/builds/234677/logs/29

File: build/azure-pipelines/sdl-scan.yml [hunk 2]
```diff
@@ -142,11 +143,15 @@ stages:
               GITHUB_TOKEN: "$(github-distro-mixin-password)"
             displayName: Download Symbols
 
+          - powershell: ls "$(agent.builddirectory)\\scanbin\\VSCode-win32-$(VSCODE_ARCH)\\pdb\\*.pdb"
```
Comment (deepak1556) [inline build/azure-pipelines/sdl-scan.yml:146]: The gulp task can instead verify the presence of pdbs and add necessary debug logging

File: build/gulpfile.scan.js (hunk 1 above)
Comment (deepak1556) [reply build/gulpfile.scan.js:49]: `swiftshader` => `vk_swiftshader` and it has corresponding dll published seems fine to remove. However we don't publish pdb for `d3dcompiler_47.dll`, should it not be removed from the exe destination to exclude from scannning ? If the Binskim task has a exclude pattern, that would be better alternative as well.

**Review by deepak1556 — COMMENTED** [review]
Can you also follow up on https://github.com/microsoft/binskim/issues/661 given it doesn't have any update for a while now. For context, chromium change to large pdb was in response to certain targets https://chromium-review.googlesource.com/c/chromium/src/+/4128591 and it will not be rolled back or can be configured by embedders. It would be good to have this addressed in Binskim.

File: build/gulpfile.scan.js [hunk 3]
```diff
@@ -106,3 +108,15 @@ function nodeModules(destinationExe, destinationPdb, platform) {
 
 	return exe;
 }
+
+function confirmPdbsExist(destinationExe, destinationPdb) {
+	readdirSync(destinationExe).forEach(file => {
+		if (file.endsWith('.dll')) {
```
Comment (deepak1556) [inline build/gulpfile.scan.js:114]: Also check for `.exe`

File: .nvmrc [hunk 4]
```diff
@@ -1 +1 @@
-18.18
\ No newline at end of file
+18.17
```
Comment (deepak1556) [inline .nvmrc:1]: Can you add a comment to not update this value until https://github.com/microsoft/vscode/issues/194921 is resolved.

Comment (rzhao271) [reply .nvmrc]: I'm unsure whether comments can be added to the .nvmrc file
The following fail:
- https://dev.azure.com/monacotools/Monaco/_build/results?buildId=235342&view=logs&j=8e733ce0-0c7b-5d06-69ef-2aad0d062863&t=20dfcc3f-29fb-5b35-1948-57062942b9b4
- https://dev.azure.com/monacotools/Monaco/_build/results?buildId=235340&view=logs&j=8e733ce0-0c7b-5d06-69ef-2aad0d062863&t=20dfcc3f-29fb-5b35-1948-57062942b9b4

Comment (deepak1556) [reply .nvmrc]: Thanks for checking

**Review by deepak1556 — CHANGES_REQUESTED** [review]
Lets' wait for https://github.com/microsoft/vscode/pull/194949 to avoid making any nvmrc changes in this PR.

**Comment by deepak1556** [comment]
Now that https://github.com/microsoft/vscode/pull/194961 is merged, should be good to go after rebase.

**Comment by rzhao271** [comment]
I rebased and squashed some commits on my side
Sanity check https://dev.azure.com/monacotools/Monaco/_build/results?buildId=235569

**Review by deepak1556 — APPROVED** [review]

**Pull request merged**